lib/conflict_free_schema/database/schema_version_files.rb

```diff
@@ -57,10 +57,15 @@ def self.init_schema_directory
 
       def self.find_versions_from_migration_files
         migration_directories.each_with_object([]) do |directory, migration_versions|
-          directory_migrations = Dir.glob(MIGRATION_VERSION_GLOB, base: directory)
-          directory_versions = directory_migrations.map! { |m| m.split('_').first }
+          directory_migrations = Dir.glob(directory)
+          directory_versions = directory_migrations.map do |migration|
+            item = Pathname.new(migration)
+            next if item.directory?
 
-          migration_versions.concat(directory_versions)
+            item.basename.to_s.match(/20[0-9][0-9]*/).to_s
```

Comment:
seems now we have handling `MIGRATION_VERSION_GLOB` constant 🤔 will take care of that post-merge tho!